src/status_im/chat/handlers/send_message.cljs

```diff
@@ -321,8 +330,9 @@
                                    :username username)))
 
                 :else
-                (protocol/send-message! (assoc-in options
-                                                  [:message :to] (:to message)))))))))))
+                (do (protocol/send-message! (assoc-in options
+                                                      [:message :to] (:to message)))
+                    (send-notification fcm-token (:message options)))))))))))
 
```

Comment:
@oskarth Do we only want to send notifications on plain, non-command messages ? Because `::send-message` handler is only used for that, commands/requests invoke `::send-command` instead.

Reply:
@janherich In a v1, yes. Later on: possibly, good point!